core/lib/multivm/src/versions/vm_latest/oracles/storage.rs

```diff
@@ -411,7 +414,7 @@ impl<S: WriteStorage, H: HistoryMode> VmStorageOracle for StorageOracle<S, H> {
                     }
                 };
 
-                let LogQuery { written_value, .. } = query.log_query;
+                let LogQuery { written_value, .. } = query.log_query.glue_into();
```

Comment:
Because the zksync_types uses the 1_3_2 ones. As per our discussion, I'll try to convert it into our custom `struct`, so that every version glues into it